resources/lang/en/metatags.php

```diff
@@ -5,17 +5,80 @@
+        'description' => 'View transactions, blocks, nodes, and other network activity on the :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
+    ],
+
+    'statistics' => [
+        'title'       => 'Statistics & Analytics - :name Blockchain Explorer',
+        'description' => 'View statistics and analyze network activity on the :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
     ],
 
     'delegates' => [
-        'title'       => 'Delegate Monitor | ARK Explorer | Cryptocurrency Block Explorer',
-        'description' => 'Monitor Delegate activity for the ARK Public Network. See Delegate rankings and track Voting Power in the ARK Blockchain.',
+        'title'       => 'Delegates - :name Blockchain Explorer',
+        'description' => 'View Delegates (Validators) and their activity on the :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
+    ],
+
+    'transactions' => [
+        'title'       => 'Transactions - :name Blockchain Explorer',
+        'description' => 'View transaction details from the :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
+    ],
+
+    'transaction' => [
+        'title'       => ':txid Transaction Details - :name Blockchain Explorer',
+        'description' => 'View information and details for transaction with ID :txid on :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
+    ],
+
+    'blocks' => [
+        'title'       => 'Blocks - :name Blockchain Explorer',
+        'description' => 'View details for blocks on the :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
+    ],
+
+    'block' => [
+        'title'       => ':blockID Block Details - :name Blockchain Explorer',
+        'description' => 'View information and details for block with ID :blockID on :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
     ],
 
     'wallets' => [
-        'title'       => 'Wallet Addresses | ARK Explorer | Cryptocurrency Block Explorer',
-        'description' => 'See wallet addresses on the ARK Explorer. Track balances and see transaction activity for wallet addresses on the ARK Public Nework',
+        'title'       => 'Wallets - :name Blockchain Explorer',
+        'description' => 'View wallet address details on the :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
+    ],
+
+    'wallet' => [
+        'title'       => ':address Wallet Details - :name Blockchain Explorer',
+        'description' => 'View balance, transaction history, and other details for :address on :name Blockchain.',
+        'image'       => asset('images/metadata/explorer-preview.png'),
+    ],
+
+    'wallet-voters' => [
+        'title'       => 'Voters of :address Wallet - :name Blockchain Explorer',
```

Comment:
Will get back to you about these delegate metadata items, as they read weird having `<address> Wallet` instead of the delegate name